packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -94,18 +94,35 @@ document.addEventListener('dart-app-ready', function (e) {
    styleSheet.parentNode.removeChild(styleSheet);
 });
 
+// Create a TrustedTypes policy so we can attach Scripts...
+var _ttPolicy;
+if (window.trustedTypes) {
+  _ttPolicy = trustedTypes.createPolicy("flutter-tools-bootstrap", {
+    createScriptURL: (url) => {
+      switch(url) {
+        case "mapper": return "$mapperUrl";
+        case "requireJs": return "$requireUrl";
```

Comment:
For the `$` string interpolation to work the string must be enclosed in backticks (``` `${requireUrl` ```) instead of double-quotes. I think the `{ }` are also required.

Reply:
Yeah, this code is confusing. The output is the following JS:

```js
createScriptURL: (url) => {
  switch(url) {
    case "mapper": return "whatever_mapper.js";
    case "requireJs": return "whatever_require.js";
```

So the double quotes are part of the JavaScript code, the String interpolation is Dart code, and it ends up hardcoding both URLs in the JavaScript output.

Reply:
I've simplified this a little bit, so we only need to use the $mapperUrl and $requireUrl once, creating a "map" of script names to their URLs.